package.json

```
@@ -25,7 +25,7 @@
     "semantic-release": "semantic-release"
   },
   "peerDependencies": {
-    "@adobe/helix-universal": "^1.0 || ^1.0.0"
+    "@adobe/helix-universal": "^1.0 || ^1.0.0 || ^1.0.0"
```

@tripodsan this does not look right.

Maybe this:
```suggestion
    "@adobe/helix-universal": "^1.0.0"
```